docs/_data/paths/seasonal-overrides_{id}.yaml

```diff
@@ -51,13 +51,12 @@ delete:
   responses:
     '204': &no-content
       $ref: '../openapi-split.yaml#/components/responses/204'
-    '304': &not-modified
-      $ref: '../openapi-split.yaml#/components/responses/304'
     '400': *malformed
     '401': *unauthorized-empty
     '403': *forbidden-empty
     '404': *not-found-empty
-    '406': *not-acceptable-empty
+    '409':
```

Comment:
When should the state of a seasonal override prevent its deletion? The only thing I can think of is if it has already been deleted, in which case this should be 410 Gone or 404 Not Found. 

Please add an example inline here (see other comment on 409) if there's a different use case.

Reply:
I would think that if you sent in to the EMS already, then you would probably want to keep it around.  I know you technically don't have the transparency requires you would with AAR exceptions- although, is that really true if the temporary seasonal rating gets used as a recourse rating?